webpack.config.js

```diff
@@ -0,0 +1,155 @@
+const webpack = require('webpack');
+const TerserPlugin = require('terser-webpack-plugin');
+const path = require('path');
+const argv = require('yargs').argv;
+const pkg = require('./package');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+
+const IS_PRODUCTION = Boolean(argv.production);
+const VERSION = argv.production ? pkg.version : 'dev';
+const SOURCE_DIR = path.join(__dirname, 'src');
+const DIST_DIR = path.join(__dirname, 'dist');
+const VERSIONED_DIST_DIR = path.join(DIST_DIR, VERSION);
+const SOURCE_DOCS_DIR = path.join(SOURCE_DIR, 'docs');
+const SOURCE_COMPONENTS_DIR = path.join(SOURCE_DIR, 'components');
+
+const babelEnv = params => [
+  '@babel/preset-env',
+  Object.assign({
+    targets: '> 0.2%, not dead, not ie < 11'
+  }, params)
+];
+
+module.exports = () => {
+  const jsEntry = path.join(SOURCE_DOCS_DIR, 'webpackEntry.jsx');
+  const jsOutput = path.join(VERSIONED_DIST_DIR, 'docs/', 'js/');
+
+  const config = {
+    target: 'web',
+    module: {
+      rules: [
+        {
+          test: /\.(ico|jpg|jpeg|png|gif|eot|otf|webp|ttf|woff|woff2|cur|ani)(\?.*)?$/,
+          loader: 'file-loader',
+          query: {
+            name: 'static/media/[name].[hash:8].[ext]'
+          }
+        },
+        {
+          test: /\.svg$/,
+          loader: 'svg-sprite-loader',
+          include: path.join(SOURCE_DIR, 'images'),
+          options: {
+            symbolId: filePath => `icon-${path.basename(filePath, '.svg')}`
+          }
+        },
+        {
+          test: /\.js$|jsx?$/,
+          loader: 'babel-loader',
+          options: {
+            cacheDirectory: true,
+            presets: [babelEnv({modules: false}), '@babel/preset-react'],
+            plugins: [
+              '@babel/plugin-transform-flow-strip-types',
+              '@babel/plugin-proposal-object-rest-spread',
+              '@babel/plugin-proposal-class-properties',
+              'react-hot-loader/babel'
+            ],
+            env: {
+              test: {
+                presets: [babelEnv({modules: 'auto'})]
+              }
+            }
+          }
+        }, {
+          test: /\.scss$/,
+          resolve: {
+            extensions: ['.scss', '.sass']
+          },
+          use: [
+            'style-loader',
+            'css-loader',
+            {
+              loader: 'postcss-loader',
+              options: {
+                ident: 'postcss',
+                plugins: [
+                  require('autoprefixer')({
```

Comment:
probably we don't need it :D

Reply:
we need it, some rules we use works under prefix even in chrome, so ui is broken without it